app/views/window/health/_patient_release_status.haml

```diff
@@ -14,4 +14,12 @@
   .alert.alert--flex.alert-danger
     %i.alert__icon.icon-warning
     %p
-      #{@patient.name} is no longer participating in the BH CP.
\ No newline at end of file
+      #{@patient.name} is no longer participating in the BH CP.
+- if @patient.coverage_level == 'none'
```

Comment:
Let's move this logic into `Health::Patient`.  It should have a `?` method to determine if it should show and a method to return the string.